demo/admin/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "private": true,
     "scripts": {
-        "admin-generator": "rimraf 'src/*/generated' && comet-admin-generator generate crud-generator-config.ts && comet-admin-generator future-generate",
+        "admin-generator": "rimraf 'src/*/generated' && comet-admin-generator generate crud-generator-config.ts && comet-admin-generator future-generate && pnpm gql:types",
```

Comment:
is that change a good idea?

Reply:
due to the admin generator is not automatically generating types for it's own, and those generated gql types get deleted everytime. I had to run this command over and over again and forgot to generate the gql:types as well.

Maybe its out of scope from this pr, i will revert this, but it's probably worth to add this later on?

Reply:
removed it.